interface/super/manage_document_templates.php

```diff
@@ -99,8 +99,32 @@
             die(xlt('Cannot determine a destination filename'));
         }
         $path_parts = pathinfo($form_dest_filename);
-        if (!in_array(strtolower($path_parts['extension'] ?? ''), array('odt', 'txt', 'docx', 'zip'))) {
-            die(text(strtolower($path_parts['extension'] ?? '')) . ' ' . xlt('filetype is not accepted'));
+        $extension = strtolower($path_parts['extension'] ?? '');
+
+        if (!in_array($extension, array('odt', 'txt', 'docx', 'zip'))) {
+            die(text($extension) . ' ' . xlt('filetype is not accepted'));
+        }
+
+        // Check if the uploaded file is a zip file
+        if ($extension === 'zip') {
+            $maxZipSize = 1048576; // 1 MB (adjust the size as needed)
```

Comment:
Is this making the max size just 1 MB ?

Reply:
(just checking; maybe is acceptable for manage_document_templates uploads)

Reply:
Yes! Making sure that if it a ZIP file, a size larger than 1MB can be denied (Although, this can be changed or even removed based on what the requirements are)